src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyAction.java

```diff
@@ -103,15 +93,21 @@ protected ActionResult execute() throws EntityDoesNotExistException {
 
             // Go to sessions page after copying,
             // so that the instructor can see the new feedback sessions
-            return createRedirectResult(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE);
+            return createAjaxResultWithoutClearingStatusMessage(
+                       new InstructorFeedbackEditCopyData(account,
```

Comment:
Weird indentation here and at `createAjaxResultWithErrorMessage`.
Do clean up this file a little.